tests/unit/backend/corpora/api_server/curator/collection/test_dataset.py

```diff
@@ -88,7 +88,7 @@ def _test(_tag, _dataset):
             self.assertIsNone(_dataset.curator_tag)
 
         dataset = self.generate_dataset(self.session, collection=collection)
-        tests = [dataset.id, "prefix" + dataset.id, dataset.id + "suffix", "prefix" + dataset.id + "suffix"]
+        tests = [dataset.id, ""]
```

Comment:
(following on from comments above) `"prefix" + dataset.id` is therefore a valid tag